tools/src/wyvern/target/corewyvernIL/support/ModuleResolver.java

```diff
@@ -207,9 +231,6 @@ public File resolve(String qualifiedName, boolean isType) {
                 names[names.length - 1] = lastName.substring(0, lastName.length() - 4) + ".wyt";
                 f = findFile2(names);
             }
-            if (f == null || !f.exists()) {
-                ToolError.reportError(ErrorMessage.MODULE_NOT_FOUND_ERROR, (FileLocation) null, isType ? "type" : "module", qualifiedName);
-            }
```

Comment:
@JonathanAldrich Does the resolve() function need to be accessed from outside ModuleResolver? If not, this function should be private till we find a way to refactor this as resolve() can return null or a non-existent file path.